lite.auto.tfvars.example

```diff
@@ -0,0 +1,254 @@
+
+# The amount of storage reserved for the landing zone in gigabytes.
+# Must be 1 or more.
+# If a storage account is defined (see below) then this value is ignored.
+# Value type: number
+# Updateable: Y
+
+storage_lz_gb=25
+
+#------------------------------------------------------------------------------
+
+# The amount of storage reserved for data in gigabytes.
+# Must be 1 or more.
+# If a storage account is defined (see below) then this value is ignored.
+# Value type: number
+# Updateable: Y
+
+storage_data_gb=100
+
+#------------------------------------------------------------------------------
+
+# Map of name => value tags that can will be associated with the cluster.
+# Format is '{"name"="value" [, "name"="value"]*}'.
+# The 'name' portion must be unique.
+# To add no tags, use '{}'.
+# Value type: map of string
+# Updateable: Y
+
+extra_tags={}
+
+#------------------------------------------------------------------------------
+
+# The VM size for each roxie node in the HPCC Systems node pool.
+# Possible values are: large, xlarge, 2xlarge, and 4xlarge
+# Value type: string
+# Updateable: N
+
+aks_roxie_node_size = "xlarge"
+
+#------------------------------------------------------------------------------
+
+# The VM size for each serv node in the HPCC Systems node pool.
+# Possible values are: large, xlarge, 2xlarge, and 4xlarge
+# Value type: string
+# Updateable: N
+
+aks_serv_node_size = "2xlarge"
+
+#------------------------------------------------------------------------------
+
+# The VM size for each spray node in the HPCC Systems node pool.
+# Possible values are: large, xlarge, 2xlarge, and 4xlarge
+# Value type: string
+# Updateable: N
+
+aks_spray_node_size = "large"
+
+#------------------------------------------------------------------------------
+
+# The VM size for each thor node in the HPCC Systems node pool.
+# Possible values are: large, xlarge, 2xlarge, and 4xlarge
+# Value type: string
+# Updateable: N
+
+aks_thor_node_size = "xlarge"
+
+#------------------------------------------------------------------------------
+
+# Email address of the administrator of this HPCC Systems cluster.
+# Example entry: "jane.doe@hpccsystems.com"
+# Value type: string
+# Updateable: Y
+
+aks_admin_email="jane.doe@hpccsystems.com"
+
+#------------------------------------------------------------------------------
+
+# Name of the administrator of this HPCC Systems cluster.
+# Example entry: "Jane Doe"
+# Value type: string
+# Updateable: Y
+
+aks_admin_name="Jane Doe"
+
+#------------------------------------------------------------------------------
+
+# Username of the administrator of this HPCC Systems cluster.
+# Example entry: "jdoe"
+# Value type: string
+# Updateable: N
+
+admin_username="jdoe"
+
+#------------------------------------------------------------------------------
+
+# The Azure region abbreviation in which to create these resources.
+# Must be one of ["eastus", "eastus2", "centralus"].
```

Comment:
We need to remove this constraint.  We have no idea which regions our users will be using.  This list was originally for when we were deploying internal clusters for our own use and the constraint no longer applies.